diff --git a/Makefile b/Makefile
index 7fdf4ec..aa00d01 100644
--- a/Makefile
+++ b/Makefile
@@ -19,10 +19,10 @@ FACILITY?=	LOG_DAEMON
 # USE_DNS to add hostname tests in hosts.allow/deny.

 ifeq ($(NO_TCP_WRAPPER),)
-CPPFLAGS += -DHOSTS_ACCESS
+override CPPFLAGS += -DHOSTS_ACCESS
 WRAP_LIB  = -lwrap
 ifdef USE_DNS
-CPPFLAGS += -DENABLE_DNS
+override CPPFLAGS += -DENABLE_DNS
 MAN_SED += -e 's/USE_DNS/yes/'
 endif
 endif
@@ -33,20 +33,20 @@ endif
 # ports. You can find out by running "rpcinfo -p"; if all mountd and NIS
 # daemons use a port >= 1024 you should probably disable the next line.

-CPPFLAGS += -DCHECK_PORT
+override CPPFLAGS += -DCHECK_PORT

 # The portmap daemon runs a uid=1/gid=1 by default.  You can change that
 # be defining DAEMON_UID and DAMEON_GID to numbers, or RPCUSER to a
 # name, though you must be sure that name lookup will not require use
 # of portmap.
 ifdef RPCUSER
-CPPFLAGS += -DRPCUSER=\"$(RPCUSER)\"
+override CPPFLAGS += -DRPCUSER=\"$(RPCUSER)\"
 MAN_SED += -e 's/RPCUSER/$(RPCUSER)/'
 else
 MAN_SED += -e 's/RPCUSER//'
 endif
 ifdef DAEMON_UID
-CPPFLAGS += -DDAEMON_UID=$(DAEMON_UID) -DDAEMON_GID=$(DAEMON_GID)
+override CPPFLAGS += -DDAEMON_UID=$(DAEMON_UID) -DDAEMON_GID=$(DAEMON_GID)
 MAN_SED += -e 's/DAEMON_UID/$(DAEMON_UID)/' -e 's/DAEMON_GID/$(DAEMON_GID)/'
 else
 MAN_SED += -e 's/DAEMON_UID/1/' -e 's/DAEMON_GID/1/'
@@ -118,14 +118,14 @@ CPPFLAGS += -DIGNORE_SIGCHLD	# AIX 4.x, HP-UX 9.x
 ### End of configurable stuff.
 ##############################

-CPPFLAGS += -DFACILITY=$(FACILITY)
-CFLAGS   ?= -O2
-CFLAGS   += -Wall -Wstrict-prototypes
+override CPPFLAGS += -DFACILITY=$(FACILITY)
+override CFLAGS   ?= -O2
+override CFLAGS   += -Wall -Wstrict-prototypes

 all:	portmap pmap_dump pmap_set portmap.man

-CPPFLAGS += $(HOSTS_ACCESS)
-portmap: CFLAGS   += -fpie
+override CPPFLAGS += $(HOSTS_ACCESS)
+portmap: override CFLAGS   += -fpie
 portmap: LDLIBS   += $(WRAP_LIB)
 portmap: LDFLAGS  += -pie
 portmap: portmap.o pmap_check.o from_local.o
--
2.7.4
